packages/devextreme/js/__internal/scheduler/appointment_popup/m_form.ts

```diff
@@ -257,6 +259,7 @@ export class AppointmentForm {
         this._dxForm = e.component;
```

Comment:
By looking at [this line](https://github.com/DevExpress/DevExtreme/pull/31555/files#diff-e2184e9321386d96f057b770cc362ad0d417f9be815ab4e9ac68f5b977f54009R224) I see that, if user has this configuration:

```js
editing.form: {
   elementAttr: {
       someAttr: 'someValue'
   }
}
```

then `elementAttr` prop will be overriden and the form element will not have `class: CLASSES.form`. This could result in a bug report in the future.

We should use a deep extend in this case. We can do that by:
```js
extend(true, defaultOptions, editingConfig.form);
```

but don't forget about onInitialized and onFieldDataChanged callbacks